src/Tracking/Repositories/Settings.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Give\Tracking\Repositories;
+
+/**
+ * Class Settings
+ * @package Give\Tracking\Repositories
+ *
+ * @since 2.10.0
+ */
+class Settings {
+	/**
+	 * Return "usage_tracking" give setting option key name.
+	 *
+	 * @since 2.10.0
+	 * @return string
+	 */
+	public function getUsageTrackingOptionKey() {
+		return 'usage_tracking';
```

Comment:
Same as my previous comment on constants. This is the same as a constant. Also for the function below.